src/Uno.UWP/UI/ViewManagement/UISettings.cs

```diff
@@ -53,13 +65,21 @@ public Color GetColorValue(UIColorType desiredColor)
 			var systemTheme = SystemThemeHelper.SystemTheme;
 			return desiredColor switch
 			{
+
+#if __MACOS__
+				UIColorType.Background => NSColor.ControlBackground,
+				UIColorType.Foreground => NSColor.ControlText,
+				UIColorType.Accent => NSColor.ControlAccent,
+#else
 				UIColorType.Background =>
```

Comment:
I am not sure if this can't cause issues - I think some people use `UIColorType.Background` and `UIColorType.Foreground` to detect the current system theme, in which case they may compare directly against `Black` and `White` 🤔

Reply:
Indeed. We only need to change Accent, not the rest for now.

Reply:
I agree that to solve the issue, Accent is enough. However, I disagree to comparing black and white colors to detect the system theme, as user may customize the system colors.